pbiviz.json

```diff
@@ -1,10 +1,10 @@
 {
   "visual": {
     "name": "SankeyDiagram",
-    "displayName": "Sankey",
+    "displayName": "Sankey 1.2.8",
     "guid": "SankeyDiagram1446463184954",
     "visualClassName": "SankeyDiagram",
-    "version": "1.2.7",
+    "version": "1.2.8",
```

Comment:
It'd be better to use `1.3.0` due to fact that this release incorporates a new feature